tests/virt/utils.py

```
@@ -355,3 +363,112 @@ def validate_machine_type(vm, expected_machine_type):
+
+
+def patch_hco_cr_with_mdev_permitted_hostdevices(hyperconverged_resource, supported_gpu_device):
+    required_keys = [MDEV_TYPE_STR, MDEV_NAME_STR, VGPU_DEVICE_NAME_STR]
+    missing_keys = [key for key in required_keys if key not in supported_gpu_device]
+    if missing_keys:
+        raise ValueError(f"Missing required keys in supported_gpu_device: {missing_keys}")
+    with ResourceEditorValidateHCOReconcile(
+        patches={
+            hyperconverged_resource: {
+                "spec": {
+                    "mediatedDevicesConfiguration": {"mediatedDeviceTypes": [supported_gpu_device[MDEV_TYPE_STR]]},
```

Pylint (3.3.7)

[convention] Line too long (115/100)

(C0301)